.netlify/build/renderers.mjs

```
@@ -0,0 +1,304 @@
+	if (typeof Component === 'object') {
+		return Component['$$typeof'].toString().slice('Symbol('.length).startsWith('react');
+	}
+	if (typeof Component !== 'function') return false;
+	if (Component.name === 'QwikComponent') return false;
+
+	// Preact forwarded-ref components can be functions, which React does not support
+	if (typeof Component === 'function' && Component['$$typeof'] === Symbol.for('react.forward_ref'))
+		return false;
+
+	if (Component.prototype != null && typeof Component.prototype.render === 'function') {
+		return React.Component.isPrototypeOf(Component) || React.PureComponent.isPrototypeOf(Component);
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'isPrototypeOf' from target object.



(lint/suspicious/noPrototypeBuiltins)